docs/readmes/orc8r/upgrade_1_6.md

```diff
@@ -0,0 +1,92 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+This guide covers upgrading Orchestrator deployments from v1.5 to v1.6. Upgrades which skip versions (e.g. v1.4 to v1.6) are not explicitly documented or supported at this time.
```

Comment:
can we say `v1.5.2 to v1.6` please? Release validation did that specifically...

Reply:
Hm. Patch versions are meant to be equivalent from this perspective. From an operator UX perspective, I think we should validate both upgrade paths -- if an operator has v1.5.0 deployed, do they have to upgrade to v1.5.2 first, before they can upgrade to v1.6.0?

From semver docs:

> PATCH version when you make backwards compatible bug fixes

> Consider a version format of X.Y.Z (Major.Minor.Patch). Bug fixes not affecting the API increment the patch version, backwards compatible API additions/changes increment the minor version, and backwards incompatible API changes increment the major version.

Realistically, we're doing semver wrong for our releases. But that aside, I think the operator UX argument is the strongest in favor of retaining the v1.5->v1.6 verbiage

Thoughts?